R/parameters.R

```diff
@@ -19,18 +19,41 @@ get_parameters <- function(iso3c = NULL,
     contact_matrix_set <- squire::get_mixing_matrix(iso3c = iso3c)
   }
 
-  c(
-    squire::parameters_explicit_SEEIR(
+  parameters <- list(
+    sq = squire::parameters_explicit_SEEIR(
```

Comment:
Why do we have to have nested parameters?

Also, I can't see where `sq` is used